srcpkgs/git-interactive-rebase-tool/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'git-interactive-rebase-tool'
+pkgname=git-interactive-rebase-tool
+version=2.1.0
+revision=1
+archs="i686 x86_64"
```

Comment:
There shouldn't be a reason to do this, but if there is, it should be clearly documented.

Reply:
@ericonr Hi, thanks for the feedback! 

What are you referring to exactly in that snippet?

Reply:
@ericonr Are you referring to the `archs` variable?

Reply:
Yes.

Reply:
Ok, thanks for the clarification. I'll push again just to update the patch. I still have to figure out the best way to disable those failing tests that the nix devs disabled.